cpp/src/strings/split/split_record.cu

```diff
@@ -0,0 +1,605 @@
+/*
```

Comment:
This is not new code. This code was simply moved out of `split.cu` in which it shared no common code.